frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Delete_HPA_Action.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Delete HPA action
+	As a user, I want to remove the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment/deployment-config 
+    And user has assigned values to cpu and memory value in resource limit section of advanced option
```

Comment:
```suggestion
     And user has added CPU resource limit
     And user has added Memory resource limit
```
Here addition of resource limit is important and not how user added the resource limits. So we can skip that part.